src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -182,6 +192,8 @@ export const ArtworkDetails: React.FC<ArtworkDetailsProps> = ({
         let nextStep = steps[nextStepIndex]
         if (nextStep === "Contact" || nextStep === "Contact Information") {
           nextRoute = `${consignPath}/${submissionId}/contact-information`
+        } else if (nextStep === "Artwork" || nextStep === "Artwork Details") {
+          nextRoute = `${consignPath}/${submissionId}/artwork-details/`
```

Comment:
Should fix the issue described and fixed here
https://github.com/artsy/force/pull/11592/files